console/src/app/services/new-auth.service.ts

```diff
@@ -19,7 +19,7 @@ export class NewAuthService {
   }
 
   public verifyMyPhone(code: string): Promise<VerifyMyPhoneResponse> {
-    return this.grpcService.authNew.verifyMyPhone({});
+    return this.grpcService.authNew.verifyMyPhone({code: code});
```

Comment:
if you want you can change it to ({ code }) but it's fine for me

Reply:
linting anyway failed so I changed it